src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -126,96 +126,98 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
         <div>
           {data.numberOfFilesAvailableForDownload > 0 && 
             <div className="DownloadListTabContent">
-              <div className="subSectionOverview">
-                <div>
-                  <div className="headlineWithHelp">
-                    <Typography variant={'headline3'}>
-                      <IconSvg options={{ icon: 'packagableFile' }} /> Web Download (.ZIP Packages)
+              <div className="subSectionOverviewContainer">
```

Comment:
Added a new wrapper div (subSectionOverviewContainer) that has the background color.  subSectionOverview still has the display: grid and is now in a bootstrap container (fixes the ticket).